controllers/admin/AdminProductsController.php

```diff
@@ -2597,7 +2597,7 @@ public function renderKpis()
         $helper->refresh = (bool)(ConfigurationKPI::get('8020_SALES_CATALOG_EXPIRE') < $time);
         $moduleManagerBuilder = new ModuleManagerBuilder();
         $moduleManager = $moduleManagerBuilder->build();
-    
+
```

Comment:
Trailling whitespaces auto corrected by my IDE.

BTW, trailling whitespaces is bad. :smile: